project-base/app/src/Controller/Admin/ProductController.php

```diff
@@ -90,14 +90,16 @@ public function catnumExistsAction(Request $request): Response
     /**
      * This route is used by GrapesJS to load Names of products
      *
-     * @Route("/product/names-by-catnums/{catnums}")
-     * @param string $catnums
+     * @Route("/product/names-by-catnums", methods={"post"}, condition="request.isXmlHttpRequest()")
+     * @param \Symfony\Component\HttpFoundation\Request $request
      * @return \Symfony\Component\HttpFoundation\JsonResponse
      */
-    public function productNamesByCatnumsAction(string $catnums): JsonResponse
+    public function productNamesByCatnumsAction(Request $request): JsonResponse
     {
+        $catnums = $request->get('catnums');
```

Comment:
I think that some validation should be done here

Reply:
What kind of validation? In theory, here may be anything